README.md

```diff
@@ -70,66 +67,6 @@ You'll need to have API access to GPT-4 to run Mentat. There are a few options t
 2. Run `export OPENAI_API_KEY=<your key here>` prior to running Mentat
 3. Place the previous command in your `.bashrc` or `.zshrc` to export your key on every terminal startup
 
-### Azure OpenAI
```

Comment:
Is any of this information anywhere else? I think it's important to keep somewhere

Reply:
It should all be here: https://docs.mentat.ai/en/latest/user/alternative_models.html

Reply:
Could we link here here? I think it's important that it's easy for people to find the docs for alternative models